### DISCUSSION TITLE
scrollContainerSelector attribute for tm-scroll message

### DISCUSSION THREAD

**Discussion by BurningTreeC** [TiddlyWiki/TiddlyWiki5#6605] [Ideas]
Hi all, I'd like to discuss #6602 which adds the `scrollContainerSelector` attribute to the tm-scroll message
In the pageScroller it detects if the scrollContainerSelector is present and it will use that as the element which gets scrolled

I think that makes the tm-scroll message more flexible

What do you think?

**Comment by linonetwo** [comment]
I didn't quite get this, you seems just replace window with scroll container, will this enable some usecases?

**Comment by Jermolene** [comment]
Hi @BurningTreeC in general discussions about a PR are best held within the PR itself, is there a reason for opening a new discussion?
